### DISCUSSION TITLE
Hi, how can I enable MSU audio the right way? I've edited the .ini file

### DISCUSSION THREAD

**Discussion by CKyle22** [snesrev/zelda3#223] [Q&A]
I've taken these steps:

Editing the .ini file and setting the flag to `true`

I created a folder in the `zelda3` folder called `msu` and put my `.pcm` files in there. I also, for good measure, put the files in `other/msu`

The thing is, the compiled program ends up being 1.8MB which is the same size as it was before I got the files and set the flag.

This can't be right, right? The files are very large!

**Comment by Nutzzz** [comment]
The .msu files are not compiled in.

The entries I use in my .ini are:
```
EnableMSU = true
MSUPath = msu/alttp_msu-
ResumeMSU = 1
MSUVolume = 25%
```
I suspect you didn't put the prefix in, e.g., my msu files are in a msu/ subdirectory named "alttp_msu-[x].pcm" where [x] is 1-34 (no leading zeros), so the MSUPath needs to reflect that part if the filenames aren't only just numbers.
